service/src/impls.rs

```diff
@@ -174,10 +201,15 @@ fn update_key(connection: &Connection, key: String, peer_id: String, timestamp_c
     };
 
     if update_allowed {
-        connection.execute(f!("
+        let mut statement = connection.prepare(f!("
              INSERT OR REPLACE INTO {KEYS_TABLE_NAME} \
-             VALUES ('{key}', '{timestamp_created}', '{timestamp_accessed}', '{peer_id}', '{pinned}', '{weight}');
-         "))
+             VALUES (?, '{timestamp_created}', '{timestamp_accessed}', ?, '{pinned}', '{weight}');
```

Comment:
why timestamp_created, timestamp_accessed, pinned & weight aren't passed via `bind`? Everything should be, I think.